show display name in title bar when user has multiple profiles

adbenitez opened the pull request:
close #3307

r10s commented on src/main/java/org/thoughtcrime/securesms/connect/DcHelper.java:
```diff
@@ -451,6 +452,10 @@ public static String getConnectivitySummary(Context context, int connectedString
       }
   }
 
+  public static String getConnectivitySummary(Context context, int connectedString) {
+    return getConnectivitySummary(context, context.getString(connectedString));
+  }
+
```
```suggestion
```

r10s commented on src/main/java/org/thoughtcrime/securesms/ConversationListActivity.java:
```diff
@@ -298,7 +298,13 @@ public void refreshTitle() {
+      if (DcHelper.getAccounts(this).getAll().length > 1) {
+        titleText = DcHelper.getConnectivitySummary(this, DcHelper.getContext(this).getName());
+      } else {
+        titleText = DcHelper.getConnectivitySummary(this, R.string.app_name);
```
```suggestion
        titleText = DcHelper.getConnectivitySummary(this, getString(R.string.app_name));
```

i think, it is easier to move `getString()` here than maintaining a second signature of `getConnectivitySummary()` - i needed a moment to understand what is going on otherwise :)

r10s replied:
ah, i see, that's a reason for two signatures then, even if minor :)

let's go with the PR as is then

r10s reviewed (approved):
some minor suggestion for the additional signature of the `getConnectivitySummary()`, but in general, nice!

adbenitez commented:
> some minor suggestion for the additional signature of the `getConnectivitySummary()`, but in general, nice!

I updated the code and removed the `getConnectivitySummary(context, int)` it was only used in two other places
